specification/recoveryservices/resource-manager/Microsoft.RecoveryServices/2016-06-01/vaults.json

```diff
@@ -50,7 +50,7 @@
         }
       }
     },
-    "/Subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.RecoveryServices/operations": {
+    "/providers/Microsoft.RecoveryServices/operations": {

```

Comment:
For this you can refer: https://github.com/Azure/azure-rest-api-specs/pull/1267 CLI is not yet released and it will be including these changes.